mkt/search/views.py

```diff
@@ -54,6 +59,25 @@ def as_view(cls, **kwargs):
         return non_atomic_requests(view)
 
 
+class MultiSearchView(SearchView):
+    serializer_class = DynamicSearchSerializer
+
+    def get_serializer_context(self):
+        context = super(MultiSearchView, self).get_serializer_context()
+        context['serializer_classes'] = {
+            'webapp': ESAppSerializer,
+            'mkt_website': ESWebsiteSerializer
```

Comment:
Yes, as I wrote that I thought about your comment when I chose the name :) It's not too late to change it so I'll do that.